cluster-up/cluster/kind-1.19-vgpu/README.md

```diff
@@ -0,0 +1,43 @@
+# K8S 1.19.0 with mdev support in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.19.0 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart. 
```

Comment:
According to the default value from https://github.com/kubevirt/kubevirtci/pull/647/files#diff-fe7cbfd7af5225b042368c965ca9c5332877d206952dc5fc483032c5a5798dfbR22
```suggestion
Provides a pre-deployed k8s cluster with version 1.19.11 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart. 
```

Reply:
done